android/KMEA/app/src/main/java/com/tavultesoft/kmea/util/WebViewUtils.java

```diff
@@ -0,0 +1,104 @@
+/**
+ * Keyman is copyright (C) SIL International. MIT License.
+ */
+package com.tavultesoft.kmea.util;
+
+import android.content.Context;
+import android.view.ViewGroup;
+import android.webkit.WebView;
+import com.tavultesoft.kmea.util.KMLog;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public final class WebViewUtils {
+  public static final String TAG = "WebViewUtils";
+
+  // Keyman Engine functionality based on Chrome version
+  public enum EngineWebViewVersionStatus {
+    DISABLED, // WebView doesn't support touch keyboard features
+    DEGRADED, // WebView supports touch keyboards but not LDML keyboards
+    FULL;     // WebView supports touch keyboards and LDML keyboards
+  }
+
+  private static final String CHROME_INSTALL_PATTERN_FORMATSTR = "^.*Chrome/([\\d\\.]+).*$";
```

Comment:
```suggestion
  private static final String CHROME_INSTALL_PATTERN_FORMATSTR = "^.*Chrome/([\\d.]+).*$";
```
?

Are Java regexes greedy by default?

Reply:
I think I read Java regex is greedy by default. My previous experience is that it also has to match the entire line.